mitxgraders/formulagrader.py

```diff
@@ -27,10 +27,6 @@
     "CalcError"
 ]
 
-def casify(thestring, case_sensitive):
-    """Converts a string to lowercase if not case_sensitive"""
-    return thestring if case_sensitive else thestring.lower()
-
 # Some of these validators are useful to other classes, e.g., IntegralGrader
 def validate_blacklist_whitelist_config(blacklist, whitelist):
```

Comment:
Line 38 has a comment about casifying. Delete it?

(Apparently I can't comment on line 38 itself since its not close enough to a change ... weird.)

Reply:
Consider it gone!